### PR TITLE
Replace function name in http trace

### PR DESCRIPTION
**Pull request by poornas**
with user friendly api prefix.

- Required for mc admin trace

## Regression
 No

## How Has This Been Tested?

with mc admin trace

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#dfa43429-aa1f-4bd5-a37e-9f4d8754bb47) |
| mint-xl.sh | :x: [more...](#ec59187f-cef3-4daa-8409-5b66f777ecba) |
| mint-large-bucket.sh | :x: [more...](#99c70eac-a2aa-4835-bd98-462a301f8ba3) |
| mint-dist-xl.sh | :x: [more...](#6f34aa18-ab88-4418-8ccc-7ec59d2e80d0) |
<a name="ec59187f-cef3-4daa-8409-5b66f777ecba"></a>
### 7794-6fce2cc/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:32609
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp fca7355378b2:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-u5a00xwdahglcv",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 3066,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:32609/aws-sdk-go-test-u5a00xwdahglcv: dial tcp 72.28.97.57:32609: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="99c70eac-a2aa-4835-bd98-462a301f8ba3"></a>
### 7794-6fce2cc/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.59:30754
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 26d98b1a4e45:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 2 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 50 seconds
(4/14) Running aws-sdk-ruby tests ... done in 15 seconds
(5/14) Running awscli tests ... done in 2 minutes and 6 seconds
(6/14) Running healthcheck tests ... done in 1 seconds
(7/14) Running mc tests ... FAILED in 2 minutes and 45 seconds
{
  "name": "mc",
  "duration": "131632",
  "function": "test_mirror_list_objects_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-26397 --quiet --no-color mirror --storage-class REDUCED_REDUNDANCY /mint/data myminio/mc-test-bucket-11225  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-11225/datafile-1-MB`\n`/mint/data/datafile-0-b` -> `myminio/mc-test-bucket-11225/datafile-0-b`\n`/mint/data/datafile-1-b` -> `myminio/mc-test-bucket-11225/datafile-1-b`\n`/mint/data/datafile-1-kB` -> `myminio/mc-test-bucket-11225/datafile-1-kB`\n`/mint/data/datafile-10-MB` -> `myminio/mc-test-bucket-11225/datafile-10-MB`\n`/mint/data/datafile-1.03-MB` -> `myminio/mc-test-bucket-11225/datafile-1.03-MB`\n`/mint/data/datafile-100-kB` -> `myminio/mc-test-bucket-11225/datafile-100-kB`\n`/mint/data/datafile-10-kB` -> `myminio/mc-test-bucket-11225/datafile-10-kB`\n`/mint/data/datafile-129-MB` -> `myminio/mc-test-bucket-11225/datafile-129-MB`\n`/mint/data/datafile-11-MB` -> `myminio/mc-test-bucket-11225/datafile-11-MB`\n`/mint/data/datafile-33-kB` -> `myminio/mc-test-bucket-11225/datafile-33-kB`\n`/mint/data/datafile-5-MB` -> `myminio/mc-test-bucket-11225/datafile-5-MB`\n`/mint/data/datafile-5243880-b` -> `myminio/mc-test-bucket-11225/datafile-5243880-b`\n`/mint/data/datafile-6-MB` -> `myminio/mc-test-bucket-11225/datafile-6-MB`\n`/mint/data/datafile-65-MB` -> `myminio/mc-test-bucket-11225/datafile-65-MB`\nmc: <ERROR> Failed to copy `/mint/data/datafile-129-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-65-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-11-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-10-MB`. Please reduce your request\nTotal: 233.17 MiB, Transferred: 560.93 MiB, Speed: 4.26 MiB/s"
}

Executed 6 out of 14 tests successfully.
```
<a name="6f34aa18-ab88-4418-8ccc-7ec59d2e80d0"></a>
### 7794-6fce2cc/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.60:32224
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 4ee7077c39d4:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 44 seconds
(4/14) Running aws-sdk-ruby tests ... done in 2 seconds
(5/14) Running awscli tests ... done in 1 minutes and 52 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 46 seconds
(8/14) Running minio-dotnet tests ... done in 39 seconds
(9/14) Running minio-go tests ... done in 49 seconds
(10/14) Running minio-java tests ... FAILED in 37 seconds
{
  "name": "minio-java",
  "function": "removeIncompleteUpload(String bucketName, String objectName)",
  "duration": 439,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code=SlowDown, message=Please reduce your request, bucketName=minio-java-test-3inm1qm, objectName=minio-java-test-3jvtj4, resource=/minio-java-test-3inm1qm/minio-java-test-3jvtj4, requestId=15A895E17207A67A, hostId=cf709da5-3245-4bc2-9632-f0aa557d4474)\nrequest={method=DELETE, url=http://72.28.97.60:32224/minio-java-test-3inm1qm/minio-java-test-3jvtj4?uploadId=6c43f129-9231-4656-a54d-2d0013522739, headers=Host: 72.28.97.60:32224\nUser-Agent: MinIO (amd64; amd64) minio-java/dev\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20190616T051426Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20190616/us-east-1/s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 361\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO/DEVELOPMENT.2019-06-16T04-55-57Z\nVary: Origin\nX-Amz-Request-Id: 15A895E17207A67A\nX-Minio-Deployment-Id: cf709da5-3245-4bc2-9632-f0aa557d4474\nX-Xss-Protection: 1; mode=block\nDate: Sun, 16 Jun 2019 05:14:26 GMT\n}\n >>> [io.minio.MinioClient.executeReq(MinioClient.java:1204), io.minio.MinioClient.execute(MinioClient.java:1070), io.minio.MinioClient.executeDelete(MinioClient.java:1347), io.minio.MinioClient.abortMultipartUpload(MinioClient.java:5149), io.minio.MinioClient.putObject(MinioClient.java:4337), io.minio.MinioClient.putObject(MinioClient.java:4183), FunctionalTest.removeIncompleteUploads_test(FunctionalTest.java:1719), FunctionalTest.runTests(FunctionalTest.java:2992), FunctionalTest.main(FunctionalTest.java:3118)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="dfa43429-aa1f-4bd5-a37e-9f4d8754bb47"></a>
### 7794-6fce2cc/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:31051
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 00b1765a0a86:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-dozdhcsg6qv2y2",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 360029,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15A895E86E5D9B8B, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#dac25a6e-9d3b-495d-b360-2d3b546bfbcd) |
| mint-xl.sh | :x: [more...](#5674b037-da18-452c-8e14-fc28a957dba7) |
| mint-dist-xl.sh | :x: [more...](#f57bfac7-4f44-4a01-827a-19ac823b7fd4) |
<a name="f57bfac7-4f44-4a01-827a-19ac823b7fd4"></a>
### 7794-1584fe2/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:32288
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 2d98a15d9640:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 43 seconds
(4/14) Running aws-sdk-ruby tests ... done in 3 seconds
(5/14) Running awscli tests ... done in 1 minutes and 54 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 53 seconds
(8/14) Running minio-dotnet tests ... done in 33 seconds
(9/14) Running minio-go tests ... done in 53 seconds
(10/14) Running minio-java tests ... FAILED in 1 minutes and 2 seconds
{
  "name": "minio-java",
  "function": "listIncompleteUploads(String bucketName, String prefix)",
  "duration": 846,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code=SlowDown, message=Please reduce your request, bucketName=minio-java-test-18b4aag, objectName=minio-java-test-1fps01i, resource=/minio-java-test-18b4aag/minio-java-test-1fps01i, requestId=15A91BB5ED8E5048, hostId=64c4e22a-cb3f-4a12-9dcb-5ca3b0e2a6bf)\nrequest={method=DELETE, url=http://72.28.97.55:32288/minio-java-test-18b4aag/minio-java-test-1fps01i?uploadId=331e82e3-e1e7-466e-8006-5ea62dfa0a92, headers=Host: 72.28.97.55:32288\nUser-Agent: MinIO (amd64; amd64) minio-java/dev\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20190617T220654Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20190617/us-east-1/s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 363\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO/DEVELOPMENT.2019-06-17T21-47-51Z\nVary: Origin\nX-Amz-Request-Id: 15A91BB5ED8E5048\nX-Minio-Deployment-Id: 64c4e22a-cb3f-4a12-9dcb-5ca3b0e2a6bf\nX-Xss-Protection: 1; mode=block\nDate: Mon, 17 Jun 2019 22:06:54 GMT\n}\n >>> [io.minio.MinioClient.executeReq(MinioClient.java:1204), io.minio.MinioClient.execute(MinioClient.java:1070), io.minio.MinioClient.executeDelete(MinioClient.java:1347), io.minio.MinioClient.abortMultipartUpload(MinioClient.java:5149), io.minio.MinioClient.putObject(MinioClient.java:4337), io.minio.MinioClient.putObject(MinioClient.java:4183), FunctionalTest.listIncompleteUploads_test2(FunctionalTest.java:1639), FunctionalTest.runTests(FunctionalTest.java:2989), FunctionalTest.main(FunctionalTest.java:3118)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="5674b037-da18-452c-8e14-fc28a957dba7"></a>
### 7794-1584fe2/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:32329
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 24bc409f243c:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-plqdruav1dlf2b",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 360011,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15A91BB546C9A050, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="dac25a6e-9d3b-495d-b360-2d3b546bfbcd"></a>
### 7794-1584fe2/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.60:30177
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 5d5eba5e0e03:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-xpy2lwkptlxqgh",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 360019,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15A91BB56FA15A02, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

File: pkg/trace/trace.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ type RequestInfo struct {
 	RawQuery string      `json:"rawquery,omitempty"`
 	Headers  http.Header `json:"headers,omitempty"`
 	Body     []byte      `json:"body,omitempty"`
+	Client   string      `json:"client"`
```
Comment (donatello) [inline pkg/trace/trace.go:41]: It looks like the `Client` field is just added here and a value passed in the other file, but the value is never read. If it is not going to be read, we don't need to add it in the struct. Can it be removed?

**Review by donatello — CHANGES_REQUESTED** [review]

File: pkg/trace/trace.go (hunk 1 above)
Comment (poornas) [reply pkg/trace/trace.go:41]: @donatello , this field is used by `mc admin trace`

Comment (donatello) [reply pkg/trace/trace.go:41]: Ah it's in the output.

**Review by donatello — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#88108323-033c-4f0f-bf09-b5b2294d6958) |
| mint-fs.sh | :x: [more...](#641364b1-82c1-482d-9ec6-7b09fe47053d) |
| mint-dist-xl.sh | :x: [more...](#0cc7e024-4a70-4980-911c-610a27aca002) |
<a name="0cc7e024-4a70-4980-911c-610a27aca002"></a>
### 7794-4b9f0f9/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.54:32714
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 423b8e98dc88:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 43 seconds
(4/14) Running aws-sdk-ruby tests ... done in 2 seconds
(5/14) Running awscli tests ... done in 2 minutes and 3 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 43 seconds
(8/14) Running minio-dotnet tests ... done in 32 seconds
(9/14) Running minio-go tests ... done in 47 seconds
(10/14) Running minio-java tests ... FAILED in 42 seconds
{
  "name": "minio-java",
  "function": "removeIncompleteUpload(String bucketName, String objectName)",
  "duration": 389,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code=SlowDown, message=Please reduce your request, bucketName=minio-java-test-91ucha, objectName=minio-java-test-1udkt73, resource=/minio-java-test-91ucha/minio-java-test-1udkt73, requestId=15A962B622CFFCD6, hostId=0aa92c9b-633f-467f-9262-3c1ec5b4ee89)\nrequest={method=DELETE, url=http://72.28.97.54:32714/minio-java-test-91ucha/minio-java-test-1udkt73?uploadId=f4ee781f-a9bf-4839-a4ac-56819975559b, headers=Host: 72.28.97.54:32714\nUser-Agent: MinIO (amd64; amd64) minio-java/dev\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20190618T194800Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20190618/us-east-1/s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 361\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO/DEVELOPMENT.2019-06-18T19-29-21Z\nVary: Origin\nX-Amz-Request-Id: 15A962B622CFFCD6\nX-Minio-Deployment-Id: 0aa92c9b-633f-467f-9262-3c1ec5b4ee89\nX-Xss-Protection: 1; mode=block\nDate: Tue, 18 Jun 2019 19:48:00 GMT\n}\n >>> [io.minio.MinioClient.executeReq(MinioClient.java:1204), io.minio.MinioClient.execute(MinioClient.java:1070), io.minio.MinioClient.executeDelete(MinioClient.java:1347), io.minio.MinioClient.abortMultipartUpload(MinioClient.java:5149), io.minio.MinioClient.putObject(MinioClient.java:4337), io.minio.MinioClient.putObject(MinioClient.java:4183), FunctionalTest.removeIncompleteUploads_test(FunctionalTest.java:1719), FunctionalTest.runTests(FunctionalTest.java:2992), FunctionalTest.main(FunctionalTest.java:3118)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="641364b1-82c1-482d-9ec6-7b09fe47053d"></a>
### 7794-4b9f0f9/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:30184
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 7a0b5a5e03db:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-yeyoiuy5q369o2",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 3055,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:30184/aws-sdk-go-test-yeyoiuy5q369o2: dial tcp 72.28.97.57:30184: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="88108323-033c-4f0f-bf09-b5b2294d6958"></a>
### 7794-4b9f0f9/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:32249
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 7f9b4f558977:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 48 seconds
(4/14) Running aws-sdk-ruby tests ... done in 14 seconds
(5/14) Running awscli tests ... done in 1 minutes and 19 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... FAILED in 2 minutes and 36 seconds
{
  "name": "mc",
  "duration": "118253",
  "function": "test_mirror_list_objects_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-8540 --quiet --no-color mirror --storage-class REDUCED_REDUNDANCY /mint/data myminio/mc-test-bucket-1951  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-1951/datafile-1-MB`\n`/mint/data/datafile-0-b` -> `myminio/mc-test-bucket-1951/datafile-0-b`\n`/mint/data/datafile-1-b` -> `myminio/mc-test-bucket-1951/datafile-1-b`\n`/mint/data/datafile-1.03-MB` -> `myminio/mc-test-bucket-1951/datafile-1.03-MB`\n`/mint/data/datafile-10-kB` -> `myminio/mc-test-bucket-1951/datafile-10-kB`\n`/mint/data/datafile-1-kB` -> `myminio/mc-test-bucket-1951/datafile-1-kB`\n`/mint/data/datafile-10-MB` -> `myminio/mc-test-bucket-1951/datafile-10-MB`\n`/mint/data/datafile-11-MB` -> `myminio/mc-test-bucket-1951/datafile-11-MB`\n`/mint/data/datafile-100-kB` -> `myminio/mc-test-bucket-1951/datafile-100-kB`\n`/mint/data/datafile-129-MB` -> `myminio/mc-test-bucket-1951/datafile-129-MB`\n`/mint/data/datafile-33-kB` -> `myminio/mc-test-bucket-1951/datafile-33-kB`\n`/mint/data/datafile-6-MB` -> `myminio/mc-test-bucket-1951/datafile-6-MB`\n`/mint/data/datafile-5243880-b` -> `myminio/mc-test-bucket-1951/datafile-5243880-b`\n`/mint/data/datafile-5-MB` -> `myminio/mc-test-bucket-1951/datafile-5-MB`\n`/mint/data/datafile-65-MB` -> `myminio/mc-test-bucket-1951/datafile-65-MB`\nmc: <ERROR> Failed to copy `/mint/data/datafile-129-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-65-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-11-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-10-MB`. Please reduce your request\nTotal: 233.17 MiB, Transferred: 572.62 MiB, Speed: 4.85 MiB/s"
}

Executed 6 out of 14 tests successfully.
```

**Pull request merged**